app-accessibility/caribou/caribou-0.4.21-r3.ebuild

```diff
@@ -0,0 +1,102 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..9} )
+PYTHON_REQ_USE="xml"
+
+inherit gnome.org gnome2-utils python-single-r1 vala
+
+DESCRIPTION="Input assistive technology intended for switch and pointer users"
+HOMEPAGE="https://wiki.gnome.org/Projects/Caribou"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+IUSE="gtk gtk2"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+
+COMMON_DEPEND="
+	${PYTHON_DEPS}
+	app-accessibility/at-spi2-core
+	$(python_gen_cond_dep '
+		>=dev-python/pygobject-2.90.3:3[${PYTHON_USEDEP}]
+	')
+	>=dev-libs/gobject-introspection-0.10.7:=
+	dev-libs/libgee:0.8
+	dev-libs/libxml2
+	>=media-libs/clutter-1.5.11:1.0[introspection]
+	x11-libs/libX11
+	x11-libs/libxklavier
+	x11-libs/libXtst
+
+	gtk? (
+		>=x11-libs/gtk+-3:3[introspection]
+	)
+	gtk2? (
+		x11-libs/gtk+:2
+	)
```

Comment:
can we drop this please

Reply:
Can you elaborate on what you mean?
* Drop the gtk2 use flag? This was added for Bug 642582, which seems like a reasonable request. Though maybe this should change to `IUSE="+gtk +gtk2"`?
* Drop gtk-2 entirely? If we're going to have gtk-3 support, we should probably still support gtk-2 as it's still maintained in-tree and there are gtk-2 only apps.
* Drop gtk[23] entirely? This goes back to my question in the PR. Is there value to having the gtk modules built when there is also direct atk support? With my limited knowledge, the gtk modules and the atk daemon seem redundant.

Reply:
For gtk2 there's tracker [bug #768993](https://bugs.gentoo.org/768993) and support removal is now done wherever possible. Not that I don't understand wanting to keep it longer considering it's for input support in other applications, but doesn't sound like a big deal to me (especially if atk works, not that I know/use this package to know what's best).